consensus/src/main/java/org/apache/iotdb/consensus/standalone/StandAloneConsensus.java

```diff
@@ -170,7 +170,8 @@ public ConsensusGenericResponse triggerSnapshot(ConsensusGroupId groupId) {
   }
 
   @Override
-  public boolean isLeader(ConsensusGroupId groupId) {
-    return true;
+  public Peer getLeader(ConsensusGroupId groupId) {
+    // StandAlone itself IS the Leader. This method should not be called
+    return null;
```

Comment:
Whether he should be returned is himself

Reply:
Indeed is, but StandAloneConsensus cannot & need not know the Endpoint(IP address), so I can't construct Myself Peer without endpoint.

Reply:
ok

Reply:
I have added the Endpoint to `standAloneConsensus` too, please wait for my [pr's](https://github.com/apache/iotdb/pull/5417) merging.

Reply:
ok